<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropOfDocDialog</class>
 <widget class="QDialog" name="PropOfDocDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Properties of Document</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="basic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Name</string>
         </property>
         <layout class="QGridLayout">
          <item row="1" column="0">
           <widget class="QLineEdit" name="title"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>URL</string>
         </property>
         <layout class="QGridLayout" name="urlGridLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="url"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>UUID</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLineEdit" name="uuid"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>KeyWords</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLineEdit" name="keywords"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Location</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLineEdit" name="location"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="opplocation"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="time">
      <attribute name="title">
       <string>Date</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Create Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="createday"/>
         </item>
         <item>
          <widget class="QTimeEdit" name="createtime"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Modified Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="modifyday"/>
         </item>
         <item>
          <widget class="QTimeEdit" name="modifytime"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Access Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="accessday"/>
         </item>
         <item>
          <widget class="QTimeEdit" name="accesstime"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>224</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="relatedocs">
      <attribute name="title">
       <string>Related Documents</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTableView" name="relatedDocs"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Protect</string>
      </attribute>
      <widget class="QCheckBox" name="hideLinkIfUnselected">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>419</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>encrypt Document</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>371</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Set Protect Password</string>
       </property>
       <widget class="QCheckBox" name="isProtect_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>125</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Protect Document</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>361</width>
          <height>132</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Original  Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Password:          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>ReInput Password:  </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="orgpwd_2">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pwd_2">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="repwd_2">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Synchronization</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>395</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Count</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QLineEdit" name="readcount"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Read Count: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Related Count</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="relatedcount"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>65</y>
         <width>395</width>
         <height>53</height>
        </rect>
       </property>
       <property name="title">
        <string>Owner</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QLineEdit" name="owner"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>7</y>
         <width>395</width>
         <height>52</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Author</string>
       </property>
       <layout class="QGridLayout" name="_3">
        <item row="1" column="0">
         <widget class="QLineEdit" name="author"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>391</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Version</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="1">
         <widget class="QLineEdit" name="version"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Local Version:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Remote Verson:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="temp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>41</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
